drivers/platform/xilinx/spi_extra.h

```diff
@@ -105,26 +105,11 @@ extern const struct spi_platform_ops spi_eng_platform_ops;
 /**
  * @brief Xilinx specific SPI platform ops structure
  */
-extern const struct spi_platform_ops xil_platform_ops;
+extern const struct spi_platform_ops xil_spi_ops;
 
 /**
  * @brief Xilinx SPI PS register functions. No extra param needed
  */
 extern const struct spi_platform_ops xil_spi_reg_ops_pl;
 
-/******************************************************************************/
-/************************ Functions Declarations ******************************/
-/******************************************************************************/
-
-/* Initialize the SPI communication peripheral. */
-int32_t xil_spi_init(struct spi_desc **desc,
```

Comment:
This part doesn't look like a rename (as stated in the commit title). That's why I asked for a commit description.
Also, were these declarations removed on purpose? The xilinx_spi.c still has the implementations.

Reply:
Those declarations were removed on purpose because the user is supposed to use those functions only via general API found in spi/spi.c. (Discussed this on Wednesday's sync meeting.)

Reply:
Got it! Thanks. But this information I believed it should be contained in the correspondent commit description.

Reply:
Will update this. Didn't want to make commit messages too long.

Reply:
Done.